isminet/models/system.py

```
@@ -0,0 +1,156 @@
+        Parameters:
+            self (SystemHealth): The SystemHealth instance being validated
+        
+        Returns:
+            SystemHealth: The validated SystemHealth instance
+        
+        Raises:
+            ValueError: If next_check is not after last_check when both timestamps are present
+        """
+        if self.last_check is not None and self.next_check is not None:
+            if self.last_check >= self.next_check:
+                raise ValueError("next_check must be after last_check")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)